xlive/Blam/Engine/rasterizer/rasterizer_settings.h

```diff
@@ -18,6 +18,15 @@ enum e_display_aspect_ratio : int32
 	_aspect_ratio_16x10 = 2
 };
 
+enum e_display_mode : int32
```

Comment:
This is just a duplicate of e_rasterizer_window_mode, e_rasterizer_window_mode is the correctly named enum.
This should be removed.